money-monkey-web/app/ui/MainSpendDisplay.tsx

```
@@ -1,37 +1,51 @@
-    total_cents: 0,
-    color: "#000000",
-  } satisfies CategorySum).total_cents;
-  const sections: CircleSection[] = sums.filter((s) => s.total_cents != 0).map(
-    ({ category_id, color, total_cents }) =>
-      ({
-        category_id: category_id,
-        color: color ?? "#0FF0AF",
-        percentage: total_cents / total,
-      }) satisfies CircleSection,
+  const total = sums.reduce(
+    (x, y) => ({ ...x, total_cents: x.total_cents + y.total_cents }),
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)